translate/api/TranslateV3Samples/TranslateV3ListGlossary.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START translate_v3_list_glossary]
+
+using Google.Api.Gax;
+using Google.Cloud.Translate.V3;
+using System;
+
+namespace GoogleCloudSamples
+{
+    public static class TranslateV3ListGlossary
+    {
+        /// <summary>
+        /// List Glossaries
+        /// </summary>
+        public static void ListGlossariesSample(string projectId)
+        {
+            TranslationServiceClient translationServiceClient = TranslationServiceClient.Create();
+            ListGlossariesRequest request = new ListGlossariesRequest
+            {
+                ParentAsLocationName = new LocationName(projectId, "us-central1"),
+            };
+            PagedEnumerable<ListGlossariesResponse, Glossary> response = translationServiceClient.ListGlossaries(request);
+            // Iterate over pages (of server-defined size), performing one RPC per page
```

Comment:
This is true, and it is what PageEnumerable does, but this behaviour is transaparent to the client code, so this comment might be confusing for someone looking at this sample.
Maybe simply describe what the foreach is doing, and then add something like "This is achieved by PagedEnumerable by iterating over pages ...."